libvmi/driver/xen/xen_events_legacy.c

```diff
@@ -119,7 +119,7 @@ static inline int get_mem_event_42(xen_mem_event_t *mem_event, mem_event_42_requ
     req_cons = back_ring->req_cons;
 
     // Copy request
-    memcpy(req, RING_GET_REQUEST(back_ring, req_cons), sizeof(*req));
+    memcpy(req, RING_GET_REQUEST(back_ring, req_cons), sizeof(mem_event_42_request_t));
```

Comment:
@tklengyel question about this (merely curious, not gating the merge): what error or warning did coverity throw?  I had previously thought that the original usage with sizeof(*blah) was actually kosher/ideal behavior from a maintenance/refactoring perspective.